lib/util/http/index.ts

```diff
@@ -1,7 +1,6 @@
 import crypto from 'crypto';
-import type { IncomingHttpHeaders } from 'http';
 import merge from 'deepmerge';
-import got, { Options, Response } from 'got';
+import got, { Options, RequestError as RequestError_, Response } from 'got';
```

Comment:
I the we don't need to rename
```suggestion
import got, { Options, RequestError, Response } from 'got';
```